scrapscript.py

```diff
@@ -1494,6 +1510,12 @@ def test_parse_double_compose(self) -> None:
             Compose(Compose(Var("h"), Var("g")), Var("f")),
         )
 
+    def test_boolean_and_binds_tighter_than_or(self) -> None:
+        self.assertEqual(
+            parse([Name("true"), Operator("||"), Name("true"), Operator("&&"), Name("false")]),
+            Binop(BinopKind.BOOL_OR, Var(name="true"), Binop(BinopKind.BOOL_AND, Var(name="true"), Var(name="false"))),
+        )
+
```

Comment:
for eval tests, we should probably test `&&` and `||` with non-bool raising

Reply:
also that they eval their args

Reply:
Can do the first part
What do you mean "that they eval their args"?

Reply:
`(1 < 2)` is an expression, not just an object, and we should test that we actually call eval on it instead of giving it literal true/false objects

you could also use variables defined in the env

Reply:
Went with the variables in the env approach